domains/pallets/operator-rewards/src/lib.rs

```diff
@@ -53,6 +55,21 @@ mod pallet {
     #[pallet::getter(fn block_rewards)]
     pub(super) type BlockRewards<T: Config> = StorageValue<_, T::Balance, ValueQuery>;
 
+    /// The domain transaction byte fee
+    ///
+    /// NOTE: we are using `ValueQuery` for convenience, which means the transactions in the domain block #1
+    // are not charged for storage fees.
```

Comment:
why? Since the domain block is derived from consensus #1, we should already get the storage fees after consensus #1, no ?

Reply:
The storage price is inserted into the domain chain through inherent extrinsic, so when validating the domain tx to be included in domain block `#1` the storage price is unknown unless we insert it into the domain genesis state.